src/jobs/database_schema_report.yml

```diff
@@ -0,0 +1,75 @@
+---
+description: >
+  Job to generate a schema report for a postgres database using schemaspy. Runs against postgres in docker
+  An HTML report is generated and is available on the artifacts tab of the circleci job
+executor: << parameters.executor_param >>
+parameters:
+  executor_param:
+    type: executor
+    default: hmpps/java_localstack_postgres
```

Comment:
We tried 
`  default:
      name: hmpps/java_localstack_postgres
      jdk_tag: "21.0"
      localstack_tag: "3.0"
      postgres_tag: "16.1"`
      
      to no avail